### PR TITLE
Using fetch api

### PR DESCRIPTION
**Pull request by imnutz**

### CODE REVIEW

File: lib/record.js [hunk 1]
```diff
@@ -163,16 +160,21 @@ exports._sendRecord = function _sendRecord (request, success, error, blockedEven
...
+          'X-TD-Write-Key': request.apikey
+        }
       }
     )
+      .then(function (res) {
+        success(res)
+      })
+      .catch(function (err) {
+        error(err)
+      })
```
Comment (tung-vu-td) [inline lib/record.js:177]: Why not simply `.then(success).catch(error)` ?

File: package.json [hunk 2]
```diff
@@ -93,15 +93,11 @@
...
-    "custom-event": "^1.0.1",
     "dayjs": "^1.8.34",
     "domready": "^0.3.0",
     "es6-promise": "^4.2.8",
     "global": "^4.3.0",
-    "json3": "^3.3.2",
-    "jsonp": "0.2.1",
-    "lit-element": "^2.3.1",
-    "lit-html": "^1.2.1",
+    "jsonp": "^0.2.1",
```
Comment (tung-vu-td) [inline package.json:100]: Why not remove jsonp?

**Review by tung-vu-td — APPROVED** [review]
LGTM

File: package.json (hunk 2 above)
Comment (imnutz) [reply package.json:100]: we still have server side cookie, which has not been implemented to support the Fetch API

**Review by nmpennypacker — APPROVED** [review]
Reviewed for Data Exfiltrations

**Review by tung-vu-td — APPROVED** [review]
LGTM

**Pull request merged**